From 3087ca3e931d29b9f3d36b251c2d79a5f644d472 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Sun, 5 Jun 2022 22:34:53 +0200
Subject: [PATCH 1/2] Change product brand from phh to google

Change product brand from phh to google can help to pass safetynet
---
 generate.sh | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/generate.sh b/generate.sh
index c1494c7..a081dd7 100644
--- a/generate.sh
+++ b/generate.sh
@@ -17,7 +17,7 @@ echo 'PRODUCT_MAKEFILES := \' > AndroidProducts.mk
 for part in a ab;do
 	for apps in vanilla gapps foss gapps-go;do
 		for arch in arm64 arm a64;do
-			for su in yes no;do
+			for su in yes no z;do
 				apps_suffix=""
 				apps_script=""
 				apps_name=""
@@ -56,6 +56,10 @@ for part in a ab;do
 					su_suffix='S'
 					extra_packages+=' phh-su me.phh.superuser'
 				fi
+				if [ "$su" == "z" ];then
+					su_suffix='Z'
+					extra_packages+='zerodaemon'
+				fi
 
 				part_suffix='a'
 				if [ "$part" == 'ab' ];then
@@ -87,8 +91,8 @@ $rom_script
 
 PRODUCT_NAME := $target
 PRODUCT_DEVICE := phhgsi_${arch}_$part
-PRODUCT_BRAND := Phh
-PRODUCT_SYSTEM_BRAND := Phh
+PRODUCT_BRAND := google
+PRODUCT_SYSTEM_BRAND := google
 PRODUCT_MODEL := Phh-Treble $apps_name
 
 PRODUCT_PACKAGES += $extra_packages
-- 
2.30.2

